jnigen/lib/src/bindings/c_bindings.dart

```diff
@@ -345,12 +349,12 @@ class CPreludes {
       '#include "dartjni.h"\n'
       '\n';
   static const defines = 'thread_local JNIEnv *jniEnv;\n'
-      'struct jni_context jni;\n\n'
-      'struct jni_context (*context_getter)(void);\n'
+      'struct JniContext jni;\n\n'
```

Comment:
I think struct can be omitted because we have 

```c
typedef struct JniContext {
	JavaVM *jvm;
	jobject classLoader;
	jmethodID loadClassMethod;
	jobject currentActivity;
	jobject appContext;
} JniContext;
```